x-pack/test/fleet_api_integration/apis/epm/final_pipeline.ts

```diff
@@ -81,6 +81,27 @@ export default function (providerContext: FtrProviderContext) {
       }
     });
 
+    it('should correctly update the final pipeline', async () => {
+      await es.ingest.putPipeline({
+        id: FINAL_PIPELINE_ID,
+        body: {
+          description: 'Test PIPELINE WITHOUT version',
+          processors: [
+            {
+              set: {
+                field: 'my-keyword-field',
+                value: 'foo',
+              },
+            },
+          ],
+        },
+      });
+      await supertest.post(`/api/fleet/setup`).set('kbn-xsrf', 'xxxx');
+      const pipelineRes = await es.ingest.getPipeline({ id: FINAL_PIPELINE_ID });
+      expect(pipelineRes.body).to.have.property(FINAL_PIPELINE_ID);
+      expect(pipelineRes.body[FINAL_PIPELINE_ID].version).to.be(1);
```

Comment:
Should this be hardcoded to 1? This test will fail when we update the `FINAL_FINAL_PIPELINE_VERSION` - maybe that's for the best?

Reply:
When we update the pipeline I think it would be nice to add other tests, but I am still moving this to a constant in this file it will make things cleaner.